configuration/config-txt/overclocking.md

```diff
@@ -10,31 +10,49 @@ Overclocking and overvoltage will be disabled at runtime when the SoC reaches 85
 
 | Option | Description |
 | --- | --- |
-| arm_freq | Frequency of the ARM CPU in MHz; default value is `1000` for Pi Zero and Pi Zero W, `700` for Pi 1, `900` for Pi 2, `1200` for Pi 3, `1400` for Pi 3A+/3B+, `1500` for Pi 4B|
-| gpu_freq | Sets `core_freq`, `h264_freq`, `isp_freq`, and `v3d_freq` together; on Pi 1/Pi 2, the default value is `250` for all items, on Pi 3/Pi Zero /Pi Zero W `core_freq` defaults to `400`, while `h264_freq`, `isp_freq`, and `v3d_freq`default to `300`; on Pi 4B the `core_freq`, `h264_freq`, `isp_freq`, and `v3d_freq` default to `500`, `600` is the only other accepted value|
-| core_freq | Frequency of the GPU processor core in MHz, influences CPU performance because it drives the L2 cache and memory bus; default value is `250` for Pi 1/Pi 2 and `400` for Pi 3/Pi Zero/Pi Zero W. For Pi 4B the default value is `500`, `600` is the only other accepted value; the L2 cache benefits only Pi Zero/Pi Zero W/ Pi 1, there is a small benefit for SDRAM on Pi 2/Pi 3 and Pi 4B|
+| arm_freq | Frequency of the ARM CPU in MHz.
+| gpu_freq | Sets `core_freq`, `h264_freq`, `isp_freq`, and `v3d_freq` together; on Pi 3/Pi Zero /Pi Zero W `core_freq` defaults to `400`, while `h264_freq`, `isp_freq`, and `v3d_freq`default to `300`; on Pi 4B this can be set to 500 or 600 only|
+| core_freq | Frequency of the GPU processor core in MHz, influences CPU performance because it drives the L2 cache and memory bus; the L2 cache benefits only Pi Zero/Pi Zero W/ Pi 1, there is a small benefit for SDRAM on Pi 2/Pi 3 and Pi 4B|
 | h264_freq | Frequency of the hardware video block in MHz; individual override of the `gpu_freq` setting |
 | isp_freq | Frequency of the image sensor pipeline block in MHz; individual override of the `gpu_freq` setting |
 | v3d_freq | Frequency of the 3D block in MHz; individual override of the `gpu_freq` setting |
-| sdram_freq | Frequency of the SDRAM in MHz; default value is `400` on Pi 1/Pi 2, `450` on Pi 3/Pi Zero/Pi Zero W, `500` on Pi 3A+/3B+, `3200` on Pi 4B. SDRAM overclocking on Pi 4B is not currently supported|
+| sdram_freq | Frequency of the SDRAM in MHz. SDRAM overclocking on Pi 4B is not currently supported|
 | over_voltage | CPU/GPU core voltage adjustment. [-16,8] equates to [0.8V,1.4V] with 0.025V steps. In other words, specifying -16 will give 0.8V as the GPU/core voltage, and specifying 8 will give 1.4V. For defaults see table below. Values above 6 are only allowed when `force_turbo` is specified: this sets the warranty bit if `over_voltage_*` is also set. |
 | over_voltage_sdram | Sets `over_voltage_sdram_c`, `over_voltage_sdram_i`, and `over_voltage_sdram_p` together. |
 | over_voltage_sdram_c | SDRAM controller voltage adjustment. [-16,8] equates to [0.8V,1.4V] with 0.025V steps. The default value is `0` (1.2V). |
```

Comment:
Should these default `over_voltage_*` values also be added to the "defaults" table below? Alternatively, perhaps `overvoltage_min` and `temp_limit` _shouldn't_ be in the "defaults" table, as they don't vary by model of Pi? (and aren't frequencies) :man_shrugging:

Reply:
Seperate table I think,  for the ones that are the same across the range. Will add.